fastlane_core/lib/fastlane_core/print_table.rb

```diff
@@ -7,7 +7,14 @@ class << self
       def print_values(config: nil, title: nil, hide_keys: [], mask_keys: [])
         require 'terminal-table'
 
-        options = config.nil? ? {} : config.values(ask: false)
+        options = {}
+        unless config.nil?
+          if config.kind_of?(FastlaneCore::Configuration)
+            options = config.values(ask: false)
+          else
+            options = config.values if options.count == 0
```

Comment:
Sure. I haven't communicated very well, sorry :smile:

1. Is the check `if options.count == 0` necessary? It feels like it will always be true at the time and place of the check.
1. Since `config` will be a Hash, `config.values` returns an Array, and `collect_rows` does `options.each do |key, value|`, is `collect_rows` intended to work with `options` that is an Array?